db/migrate/20160407121458_remove_redundant_policy.rb

```diff
@@ -0,0 +1,15 @@
+class RemoveRedundantPolicy < ActiveRecord::Migration
+  # Remove /government/policies/schools-and-college-qualifications-and-curriculum
+  def up
+    content_items = ContentItem.where(content_id: "3c04de88-9e4a-4ebb-bdc9-ef5946db17b9")
```

Comment:
Might be a good idea to wrap this in a transaction in case something goes wrong half way through, or are migrations automatically wrapped? 

Also doesn't look like it removes the content_items?